collect_app/src/main/java/org/odk/collect/android/utilities/ListViewUtils.java

```diff
@@ -57,4 +60,18 @@ public static void setAllToCheckedState(ListView lv, boolean check) {
             lv.setItemChecked(x, check);
         }
     }
+
+    // Function to toggle button label
+    public static void toggleButtonLabel(Button mToggleButton, ListView lv) {
+        if (ListViewUtils.selectedItemCount(lv) != lv.getCount())
```

Comment:
The Android style guide does not accept conditionals without braces like that. Search for "brace style" on [this page](https://source.android.com/source/code-style.html). The reason is that it's easy to introduce subtle bugs by adding lines of code but forgetting to add in the braces.